src/libPMacc/include/eventSystem/tasks/StreamTask.hpp

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2013 Felix Schmitt, Rene Widera
+ * Copyright 2013-2015 Felix Schmitt, Rene Widera, Benjamin Worpitz
```

Comment:
This slipped through because I did not include the other changes to this PR. Will be removed.